docs/CHANGELOG.asciidoc

```diff
@@ -68,6 +68,453 @@ Review important information about the {kib} 7.x releases.
 
 coming::[7.15.0]
 
+For information about the {kib} 7.15.0 release, review the following information.
+
+[float]
+[[known-issue-7.15.0]]
+=== Known issue
+
+.Upgrade Elastic Agents to use Osquery Manager integration
+[%collapsible]
+====
+*Details* +
+You must upgrade your Elastic Agents to the latest version to use the Osquery Manager integration.
+
+*Impact* +
+To upgrade, refer to {fleet-guide}/upgrade-elastic-agent.html[Upgrade Elastic Agent].
+====
+
+[float]
+[[breaking-changes-7.15.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and performance.
+Before you upgrade to 7.15.0, review the breaking changes, then mitigate the impact to your application.
+
+// tag::notable-breaking-changes[]
+
+[discrete]
+[[breaking-108961]]
+.Updating UI to work with new kibana privileges abstraction on alerts
+[%collapsible]
+====
+*Details* +
+!!TODO!! For more information, refer to {kibana-pull}108961[#108961]
+
+*Impact* +
+!!TODO!!
+====
+      
+[discrete]
+[[breaking-104196]]
+.Remove metadata query strategy v1
+[%collapsible]
+====
+*Details* +
+!!TODO!! For more information, refer to {kibana-pull}104196[#104196]
+
+*Impact* +
+!!TODO!!
+====
+      
+[discrete]
+[[breaking-103652]]
+.ML locator
+[%collapsible]
+====
+*Details* +
+!!TODO!! For more information, refer to {kibana-pull}103652[#103652]
+
+*Impact* +
+!!TODO!!
+====
+      
+[discrete]
+[[breaking-103494]]
+.Create expression functions for object types used in agg params
+[%collapsible]
+====
+*Details* +
+!!TODO!! For more information, refer to {kibana-pull}103494[#103494]
+
+*Impact* +
+!!TODO!!
+====
+
+[discrete]
+[[breaking-tbd]]
+.New scheduled query action ID format
```

Comment:
It might be good to make it clear in the title that this relates to Osquery Manager (there is also an osquery Filebeat module and a log collection integration). Maybe something as simple as:

> New scheduled query action ID format for Osquery Manager

or

> Osquery Manager: New scheduled query action ID format

Something like that?